frontend/src/components/pages/profile/components/friendsList/friendsList.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../../../styles/variables.scss';
+
+.friends{
+  @include tile;
+  margin-top: $gap-24;
+
+  @include for-mobile{
+    width: $mobile-tile-width;
+  }
+
+    .title{
+      @include title;
+      position: relative;
+      margin: $gap-4;
+      text-decoration: none;
+      color: $black;
+      width: 100%;
+
+    span{
+      @include text;
+      color: $gray;
+      margin-left: $gap-4;
+    }
+  }
+
+  .friends-list{
+     display: flex;
+     justify-content: space-between;
+     flex-wrap: wrap;
+
+     .friend{
+       position: relative;
+       flex-basis: 30%;
+       text-align: center;
+       text-decoration: none;
+       color: $black;
+       margin-top: $gap-20;
+
+       .user-photo{
```

Comment:
you do not have that class in that component. if you want to make changes to child component - it is better to do via props/flags. So all logic will be encapsulated in one component. So if you want to make photo small or big, add such props